resources/examples/Showtimes/blueprints/changelog-public-only-all-capabilities.md

```diff
@@ -21,41 +22,41 @@ Changed up the responses for `/movie/{id}`, `/movies/{id}` and `/movies`.
 ### Reference
 #### Changed
 ##### Resources
-- `/movies/{id}` now returns a `application/mill.example.movie` Content-Type header on the following HTTP methods:
-    - `GET`
-    - `PATCH`
-- `/movies` now returns a `application/mill.example.movie` Content-Type header on the following HTTP methods:
-    - `GET`
-    - `POST`
-- `/theaters/{id}` now returns a `application/mill.example.theater` Content-Type header on the following HTTP methods:
-    - `GET`
-    - `PATCH`
-- `/theaters` now returns a `application/mill.example.theater` Content-Type header on the following HTTP methods:
-    - `GET`
-    - `POST`
+- The following Movies resources have changed:
```

Comment:
I should designate "Movies" with the templating system I have so in Markdown this'll be wrapped in backticks, and in JSON, a `mill-changelog` styleable `<span>` element.